### PR TITLE
[changelog] PostgreSQL 12.5.0-1, 11.10.0-1, 10.15.0-1 and 9.6.20

### PR DESCRIPTION
**Pull request by EtienneM**
Tweet:

> [Changelog] - PostgreSQL - New default version: 12.5.0-1 - https://changelog.scalingo.com #postgresql #changelog #PaaS

### CODE REVIEW

File: changelog/databases/_posts/2020-11-16-postgresql-12.5.0-1.markdown
Comment (Soulou) [inline changelog/databases/_posts/2020-11-16-postgresql-12.5.0-1.markdown:11]: Actually in contains fixes for several CVE, maybe linking https://www.postgresql.org/about/news/postgresql-131-125-1110-1015-9620-and-9524-released-2111/ is better?

**Review by Soulou — CHANGES_REQUESTED** [review]

File: changelog/databases/_posts/2020-11-16-postgresql-12.5.0-1.markdown
Comment (Soulou) [reply changelog/databases/_posts/2020-11-16-postgresql-12.5.0-1.markdown:11]: And mentioning all databases will be updated in the coming days

**Review by Soulou — APPROVED** [review]
LGTM

**Comment by EtienneM** [comment]
Did you kept it as is on purpose or can I merge it?

**Comment by Soulou** [comment]
@EtienneM after un LGTM you can merge if all checks are green ;)

**Pull request merged**
